src/GetComments.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+
+export default function GetComments(props) {
+  const { article_id } = props;
+  const [comments, setComments] = useState([]);
+  const [commentsHidden, setCommentsHidden] = useState(true);
+
+  useEffect(() => {
+    fetch(
+      `https://nc-news-itve.onrender.com/api/articles/${article_id}/comments`
```

Comment:
I think you mentioned you're going to export these later and change them to axios, but just thought I'd leave a gentle reminder here! 😄